### DISCUSSION TITLE
Filter/aggregate source from related domains

### DISCUSSION THREAD

**Discussion by espen** [plausible/analytics#803] [Anything Else]
I run a SaaS and have many links to my website from client accounts URLs. Basically: client1.example.org, client2.example.org etc show up as sources on the website example.org. It's nice to see this but given that there are hundred of such client accounts and that there are quite many clicks it clutters up the source stats. What would you recommend for this scenario? Should I add utm-info to the links on client.example.org? Is there any way to aggregate clicks from all subdomains, but still be able to see totals for each subdomain?

**Comment by metmarkosaric** [comment]
Thanks @espen! By default the Top Sources will surface the most important referrals as it's ranked by number of clicks they send. But if you want your clients not to clutter your Top Sources, the best would be to use UTM tags on their links to you. You can still get the clicks from individual client by using a different UTM_campaign. So something like this:

yourdomain.com?utm_source=Clients&utm_medium=Websites&utm_campaign=Client1
yourdomain.com?utm_source=Clients&utm_medium=Websites&utm_campaign=Client2
etc

**Reply by espen** [reply]
Thanks, so this will remove the clicks from URL sources? So instead of client1.example.org, client2.example.org it will show as utm_source 'clients'?

**Reply by metmarkosaric** [reply]
Yes. When a UTM parameter is present in the link it takes priority over the referrer header. You can read more in our docs: https://plausible.io/docs/manual-link-tagging

**Reply by espen** [reply]
Ah, I didn't see that doc. Thanks.

**Reply by metmarkosaric** [reply]
you're welcome!
